versioneer.py

```diff
@@ -0,0 +1,10 @@
+from pkg_resources import resource_stream
+
```

Comment:
versioneer is usually a stand alone thing that is its own beast of a module?

Reply:
Yup. Didn't feel like dealing with the beast right now or figuring out how to resolve that with the version in [microsetta_public_api.yml](https://github.com/biocore/microsetta-public-api/pull/1/files#diff-bbe3166f15072419a91a31b142451247) so I just added a minimalist one for now.
See #2